android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/datatypes/VideoStreamingCapabilityTests.java

```diff
@@ -68,11 +71,15 @@ public void testRpcValues() {
         assertNull(TestValues.NULL, msg.getDiagonalScreenSize());
         assertNull(TestValues.NULL, msg.getPixelPerInch());
         assertNull(TestValues.NULL, msg.getScale());
+        assertNull(TestValues.NULL, msg.getAdditionalVideoStreamingCapabilities());
     }
 
     public void testJson() {
         JSONObject reference = new JSONObject();
 
+        msg.setAdditionalVideoStreamingCapabilities(null);
```

Comment:
@santanamk

The "testJson" throws an exception while serializing using raw serializeJSON method, though while using it in real life everything goes fine. This is produced because of the different serializeJSON(byte protocolVersion) method and additional logic it has inside. If we put there  `reference.put(VideoStreamingCapability.KEY_ADDITIONAL_VIDEO_STREAMING_CAPABILITIES...)` it will constantly lead to the stackOverflowExceptions. On the other hand - I was unable to provide some substitution for this. 

> Original reply